### DISCUSSION TITLE
Security link not scrolling to top of page in Firefox

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#18154] [Contribute to Supabase]
This link towards the bottom of supabase.com links to the supabase.com/security page. 
![screenshot-2023-10-12-at-15 34 33](https://github.com/supabase/supabase/assets/105593/b26aed72-334f-4459-b483-d61d0c98f7ae)

On firefox, when you click , it goes to that page but maintains the scroll position from the homepage. Its a new route, so it should scroll to the top. 

<img width="1450" alt="screenshot-2023-10-12-at-15 36 35" src="https://github.com/supabase/supabase/assets/105593/1216fbdb-0e8d-4b0f-9c77-9f3d8e36c2cf">

**Comment by benmann** [comment]
Same actually goes for links under 'Product' and other categories in the footer, and potentially other links in general. This is because they're rendered as `<Link>`s and not real `<a>` tags (based on [this loop](https://github.com/supabase/supabase/blob/master/apps/www/components/Footer/index.tsx#L131) and the data not starting with `https`). It appears the `scroll-behavior: smooth !important;` on the html tag is not playing nice with next/link in Firefox. Seems to be a [somewhat known ](https://github.com/vercel/next.js/issues/12105) issue. Two possible solutions:

1) Trick the link into acting like a dynamic route + empty hash, e.g. `<Link href="#" as="/security">`
2) Remove the `scroll-behavior` ideally for Firefox only, through a media query, e.g.
```
@-moz-document url-prefix() {
  html {
    scroll-behavior: auto !important;
  }
}
```

The first solutions isn't super reliable and feels a bit hacky, so I'd vote for 2) personally, even though it does remove the smooth scroll effect for FF users (I wouldn't expect it anyway tbh. as it's a full page transition in these cases).

**Comment by jacobabahn** [comment]
Could I get this issue? It'd be my first one and I'd like to contribute!

**Comment by fsansalvadore** [comment]
I can't replicate this on my end on Firefox.
Are you still experiencing this @saltcod?
